### PR TITLE
[User Accounts] Make Menu Filter use the white space

### PR DESCRIPTION
**Pull request by johnsaigle**
# Before 

screen shot 2017-07-20 at 4 01 14 pm

---

# After
screen shot 2017-07-20 at 4 00 58 pm

### CODE REVIEW

File: modules/user_accounts/templates/menu_user_accounts.tpl [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 <!-- start the selection table -->
 <div class="row">
-    <div class="col-sm-9 col-lg-8">
+    <div class="col-sm-9 col-lg-12">
```
Comment (jstirling91) [inline modules/user_accounts/templates/menu_user_accounts.tpl]: Check with others about the look and feel of this. I'm not the biggest fan of the way it looks. I feel like we could use the white space in a better way. @taracampbell can you have a look at this. If people seem good with the way it looks then change this line to be `class="col-xs-12"`

Comment (johnsaigle) [inline modules/user_accounts/templates/menu_user_accounts.tpl]: Yeah I'm happy to have input on the design decisions. I just think it looks bad/weird the way it is but I'm not married to any one solution.

Comment (taracampbell) [inline modules/user_accounts/templates/menu_user_accounts.tpl]: I think the 1/3 block of white space we have currently is a bit weird for sure. I don't see us putting anything else in that space, although we may still have something in candidate list where you can enter the PSCID/DCCID to open a candidate? I can't remember if we removed that or not.

I prefer the full width idea for now. And hopefully we can further revamp filters in the near future.

Comment (johnsaigle) [inline modules/user_accounts/templates/menu_user_accounts.tpl]: I think I know what you're talking about but I don't think it exists in `user_accounts`. Definitely lies somewhere else though.

Comment (taracampbell) [inline modules/user_accounts/templates/menu_user_accounts.tpl]: @johnsaigle oh ya whoops, thought this was a general change. Shouldn't we change all of them at the same time?

**Comment by johnsaigle** [comment]
@jstirling91 can you update your review?

**Comment by jstirling91** [comment]
A design decision needs to be made about if we want to get rid of the white space on the right of the selection filters. If so we should do this to all modules.

**Pull request closed without merging**